src/django/users/models.py

```diff
@@ -116,6 +116,9 @@ def create_superuser(self, email, password, **extra_fields):
 
         return self._create_user(email, password, **extra_fields)
 
+    def get_root_user(self):
+        return self.get(email='systems+pfb@azavea.com')
```

Comment:
I considered this, but I ran out of time given this was a can of worms with edge cases that I'd rather not open at this time. The root user _should_ be the first admin user, but it's possible that it could end up not being so at some point. 

I think a decent additional step would be to codify the root user email in a settings variable as you suggested. That could be done later, I figured encapsulating access to the root user object via this method was a decent stopgap for now that at least removed duplicate email strings scattered throughout the code.